buyTogetherApp/templates/buyTogetherApp/main.html

```diff
@@ -0,0 +1,131 @@
+                width: 100%;
+              }
+              
+              td, th {
+                border: 1px solid #dddddd;
+                text-align: left;
+                padding: 8px;
+              }
+              
+              tr:nth-child(even) {
+                background-color: #dddddd;
+              }
+              .projectName{
+                  text-align: center;
+                  font-size: 100px;
+                  color: purple;
+              }
+              .bigFont{
+                  font-size: 30px;
+                  margin-top: 80px;
+                  margin-left: 20px;
+                  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+              }
+              .red{
+                  color: red;
+              }
+            .blue{
+                color: blue;
+            }
+
+        </style>
+    </head>
+
+    <body>
+        <div class="projectName red">
+            buyTogether
+        </div>
+        <div class="bigFont">
+            <p>The idea of the project is to let group of people to organize together and buy straight from the supplier at lower cost.</p>
+            Goals
+            <br>
+            - to create simple and easy communication between an organized group of people and a supplier.
+            <br>
+            - lower the prices of food cost
+            <br>
+            Our plan is to create a website that will communicate between the group of people and the supplier, removing the supermarkets from the equation, which will lead to lower the price for the customer.
+        </div>
+        <br>
+        <table>
+            <tr>
+              <th>Name</th>
+              <th>Year</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>Elipaz Menachem</td>
+              <td>28 years old</td>
+              <td>third year of computer science degree</td>
+            </tr>
+            <tr>
+              <td>Lior Silberman</td>
+              <td>27 years old</td>
+              <td>I'm a third year computer science student</td>
+            </tr>
+            <tr>
+              <td>Meitar Rizner</td>
+              <td>26 years old</td>
+              <td>A third year computer science student</td>
+            </tr>
+            <tr>
+              <td>Rawan Abuleil</td>
+              <td>23 years old</td>
+              <td>third year of computer science degree</td>
+            </tr>
+            <tr>
+              <td>Karam Jbareen</td>
+              <td>23 years old</td>
+              <td>fourth year of computer science degree</td>
+            </tr>
+          </table>
+          <p class="bigFont">
+              We will all be working and helping each other in each field and each division but basically ,2 will be working on the front end and 2 on the back end and one on the communication between the group and QR checking codes and all together will be managing the database and models.
```

Comment:
You might recall it was mentioned on the lectures and also on our group's Slack channel earlier this week, that we're aiming towards vertical work division (vertical integration), in which features are designed and created small-enough so they are added top-to-bottom.

This means that a single team member owns all aspects of a feature, rather than some being in charge of the front-end aspects of the feature and others on the its back-end aspects.

For this demo, you should think together as a group and decide in high-level what are the features you want introduced -- each of those you should be able, eventually, to break down into small-enough self-contained features that can be owned by a single member.

Reply:
We will be working at these things in today's meeting

Reply:
Done